extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -213,6 +218,8 @@ class AmpAccordion extends AMP.BaseElement {
 
   /**
    * Toggles section between expanded or collapsed.
+   * @param {!Element} section
+   * @param {boolean=} opt_forceExpand
    * @private
    */
   toggle_(section, opt_forceExpand = undefined) {
```

Comment:
cleanup nit: unspecified arguments are `undefined` by default, so it shouldn't need to be explicitly stated.